features/finitio/language/parametric_polymorphism/binding_aliases.feature

```diff
@@ -0,0 +1,42 @@
+Feature: Support for subtyping a parametric type by binding its parameters with alias syntax
+
+  Background:
+
+    Given the System is
+      """
+      TakesTwo(first, second) = {
+        fieldOne : first,
+        fieldTwo : second
+      }
+      BindsTwo(first) = TakesTwo(first, Boolean)
+      BindsTwo(Integer)
+      """
+
+  Scenario: Validating data against valid document
+
+    Given I dress the following JSON document:
+      """
+      {
+        "fieldOne": 5,
+        "fieldTwo": true
+      }
+      """
+
+    Then it should be a success
+
+  Scenario: Validating data against an invalid document (I)
+
+    Given I dress the following JSON document:
+      """
+      {
+        "fieldOne": 5,
+        "fieldTwo": "foo"
+      }
+      """
+
+  Scenario: Figure out how to assert that BindsTwo is a subtype of TakesTwo
+
+    Then it should be a TypeError
+    And its root cause should be:
+      | message                |
+      | Invalid Boolean: `foo` |
```

Comment:
I've never used cucumber, so I'm not sure. On second thought, it probably isn't a good idea to make a partially bound type a subtype of the completely unbound version, so I'll just remove the second test when I go through to rename the test files.